library/src/scala/internal/quoted/Unpickler.scala

```diff
@@ -7,7 +7,7 @@ object Unpickler {
 
   type PickledQuote = List[String]
   type PickledExprArgs = Seq[Seq[Any] => ((QuoteContext ?=> Expr[Any]) | Type[_])]
-  type PickledTypeArgs = Seq[Seq[Any] => Type[_]]
+  type PickledTypeArgs = Seq[Seq[Any] => ((QuoteContext ?=> Expr[Any]) | Type[_])]
```

Comment:
Can we simplify to only keep `PickledArgs`, as `PickledExprArgs` and `PickledTypeArgs` now have the same definition.

Reply:
Yes, but there are some bootstrapping constraints.